Cilsil/Program.cs

```diff
@@ -154,13 +159,17 @@ private static void Main(string[] args)
                 PrintCfg(cfg, printprocs);
             }
 
-            watch.Stop();
-
             Log.PrintAllUnknownInstruction();
             Log.WriteLine();
             Log.PrintCoverageStats(result.GetResult<CfgParserResult>().Methods);
-            Log.WriteLine();
-            Log.PrintProcessTime(watch.ElapsedMilliseconds);
+            
+            watch.Stop();
+
+            if (!string.IsNullOrWhiteSpace(outelapsetime))
```

Comment:
use Microsoft.FeatureManager which is directly targeting this scenario rather than directly comparing string using if

Reply:
Microsoft.FeatureManager is for more complicated case. I update with one debug flag